common/src/main/java/nearsoft/skytouch/common/model/Product.java

```diff
@@ -0,0 +1,32 @@
+package nearsoft.skytouch.common.model;
+
+public class Product {
```

Comment:
Try to always override `toString` method.

Reply:
wouldn't better applying the change now?

Reply:
since the comment was made in the top of the class when i add the change it had no affect on the commen i think, as you can see in https://github.com/JxJxJxMxNxS/skt-task/commit/332934e8c50f6d5406fc09404d0b6f0dfb7e3fe0 i made the requested changes